python/tests/conftest.py

```diff
@@ -52,55 +54,70 @@ def _get_test_data() -> TTestData:
 
 
 @pytest.fixture(scope="function")
-def remove_test_dashboards(sdk: mtds.LookerSDK, dashboards):
+def remove_test_dashboards(sdk40: methods40.Looker40SDK, dashboards):
     # Clean up any test dashboards that may exist. We do this at the beginning
     # instead of the end of tests in case we want to view the dashboards after the test
     for d in dashboards:
-        search_results = sdk.search_dashboards(title=d["title"])
+        search_results = sdk40.search_dashboards(title=d["title"])
         if len(search_results) > 0:
             for dashboard in search_results:
-                sdk.delete_dashboard(cast(str, dashboard.id))
+                sdk40.delete_dashboard(cast(str, dashboard.id))
```

Comment:
if dashboard.id can be a LookML dashboard ID ref, I don't know why we still have integer dashboard.id for this structure. We should fix this for 4.0 or someone won't be able to do anything with LookML dashboards?

Reply:
This is just a typing cast, so it returns the `dashboard.id` unchanged but signals to `mypy` that the return value is a `str`. The reason it's needed is because `search_dashboards` returns a `models.Dashboard` where `id` is of type `Optional[str]`. Without the cast `mypy` complains that we might be passing a `None` type when a `str` is expected to `delete_dashboard`.